patches/minecraft/net/minecraft/command/Commands.java.patch

```diff
@@ -15,3 +15,11 @@
              return this.field_197062_b.execute(stringreader, p_197059_1_);
           } catch (CommandException commandexception) {
              p_197059_1_.func_197021_a(commandexception.func_197003_a());
+@@ -273,6 +281,7 @@
+                argumentbuilder.executes((p_197053_0_) -> {
+                   return 0;
+                });
++               argumentbuilder.executes(null);// FORGE Client Commands support
```

Comment:
What does this do? Why is this needed for client commands to work? Please add some more description to the comment

Reply:
This is explained in the commit message:
"Patch Commands to remove empty lambdas of server commands from being
sent to client (and thus execute there when they should not)."
The line right above this is the one where "empty lambdas" are set into the builder and then sent to the client by SCommandListPacket. They will then get registered by CommandDispatcher, and would be trapped by the hook of ForgeHooksClient in ChatScreen.
Without this patch, server commands are executed as client commands which do nothing since empty lambdas are always successful, effectively trapping all server commands on the client side.

Reply:
I somewhat understand this, but please add a brief version of this to the code comment so that when someone needs to port patches they understand why this is there as well